packages/react-notion-x/src/third-party/collection-view-calendar.tsx

```diff
@@ -0,0 +1,436 @@
+          </div>
+
+          <div
+            style={{
+              flexGrow: 1
+            }}
+          ></div>
+
+          {showCalendarControls && (
+            <>
+              <div
+                className={cs(
+                  'notion-focusable',
+                  'notion-calendar-header-inner-controls-prev'
+                )}
+                role='button'
+                tabIndex={0}
+                onClick={prevMonth}
+              >
+                <SvgLeftChevron className='notion-calendar-header-inner-controls-prev-svg' />
+              </div>
+
+              <div
+                className={cs(
+                  'notion-focusable',
+                  'notion-calendar-header-inner-controls-today'
+                )}
+                role='button'
+                tabIndex={0}
+                onClick={nowMonth}
+              >
+                Today
+              </div>
+              <div
+                className={cs(
+                  'notion-focusable',
+                  'notion-calendar-header-inner-controls-next'
+                )}
+                role='button'
+                tabIndex={0}
+                onClick={nextMonth}
+              >
+                <SvgRightChevron className='notion-calendar-header-inner-controls-next-svg' />
+              </div>
+            </>
+          )}
+        </div>
+
+        <div className='notion-calendar-header-days'>
+          {startWeekOnMonday ? (
+            <>
+              <div className='notion-calendar-header-days-day'>Mon</div>
+              <div className='notion-calendar-header-days-day'>Tue</div>
+              <div className='notion-calendar-header-days-day'>Wed</div>
+              <div className='notion-calendar-header-days-day'>Thu</div>
+              <div className='notion-calendar-header-days-day'>Fri</div>
+              <div className='notion-calendar-header-days-day'>Sat</div>
+              <div className='notion-calendar-header-days-day'>Sun</div>
+            </>
+          ) : (
+            <>
+              <div className='notion-calendar-header-days-day'>Sun</div>
+              <div className='notion-calendar-header-days-day'>Mon</div>
+              <div className='notion-calendar-header-days-day'>Tue</div>
+              <div className='notion-calendar-header-days-day'>Wed</div>
+              <div className='notion-calendar-header-days-day'>Thu</div>
+              <div className='notion-calendar-header-days-day'>Fri</div>
+              <div className='notion-calendar-header-days-day'>Sat</div>
+            </>
+          )}
+        </div>
+      </div>
+
+      <div
+        style={{
+          height: '66px'
+        }}
+      ></div>
+
+      <div className='notion-calendar-body'>
+        {weeksArr.map((i, indexI) => (
+          <div
+            className={cs(
+              'notion-calendar-body-inner',
+              weeksArr.length - 1 == indexI &&
```

Comment:
This repo uses eslint to enforce `===` over `==` for safety reasons.

Reply:
There are lots of cases of this; I'm not going to comment on them, but running `eslint` and `prettier` locally should call them out for you.